init_database.py

```
@@ -0,0 +1,172 @@
+    for db_config in DB_CONFIGS:
+        collection_name = db_config["collection"]
+        print(f"\n📦 コレクション: {collection_name}")
+
+        try:
+            # ChromaDBに接続
+            db = SongVectorDB(
+                collection_name=collection_name, distance_fn="cosine", use_remote=True
+            )
+
+            # 全曲を取得（最大100万曲）
+            print(f"  楽曲データを取得中...")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)